File: mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(void);

int main(void)
{
    int h = get_height();
    
    for (int i=0; i<h; i++)
    {
        for(int j=h-1-i; j>0; j--)
            printf(" ");
        for(int k=0; k<i+1; k++)
            printf("#");
        printf("\n");
    }
}

int get_height(void)
{
    int n;
    do
    {
        n = get_int("Height : ");
    }
    while(!(n>=1&&n<=8));
    return n;
}

File: mario.py
from cs50 import get_int

def main():
    size = get_input()

    for i in range(1, size + 1):
        print(" "*(size - i) + "#"*i + "  " + "#"*i)


def get_input():
    while True:
        n = get_int("Height: ")
        if n in range(1,9):
            break
    return n

main()